arangod/RocksDBEngine/RocksDBZkdIndex.cpp

```diff
@@ -579,10 +578,36 @@ auto zkd::supportsFilterCondition(
     return {};  // all prefix values have to be assigned
   }
 
-  // TODO -- actually return costs
-  auto costs = Index::FilterCosts::defaultCosts(itemsInIndex);
-  costs.coveredAttributes = extractedBounds.size() + extractedPrefix.size();
+  Index::FilterCosts costs;
   costs.supportsCondition = true;
+  costs.coveredAttributes = extractedBounds.size() + extractedPrefix.size();
+
+  // we look up a single point using the prefix values
+  const double estimatedElementsOnCurve =
+      index->hasSelectivityEstimate() ? 1. / index->selectivityEstimate()
```

Comment:
The selectivity estimate could be `0` here, so we would attempt to divide by zero.